openvdb/points/AttributeArray.h

```diff
@@ -1172,7 +1188,9 @@ TypedAttributeArray<ValueType_, Codec_>&
 TypedAttributeArray<ValueType_, Codec_>::operator=(const TypedAttributeArray& rhs)
 {
     if (&rhs != this) {
+        // lock both the source and target arrays to ensure thread-safety
         tbb::spin_mutex::scoped_lock lock(mMutex);
+        tbb::spin_mutex::scoped_lock rhsLock(const_cast<tbb::spin_mutex&>(rhs.mMutex));
```

Comment:
Rather than have to cast away constness, perhaps make `mMutex` mutable?

Reply:
Done, thanks.